app/src/main/java/com/sunasterisk/youtubebackground/screen/fragment/home/HomeFragment.kt

```diff
@@ -0,0 +1,24 @@
+package com.sunasterisk.youtubebackground.screen.fragment.home
+
+import android.os.Bundle
+import android.view.LayoutInflater
+import android.view.View
+import android.view.ViewGroup
+import androidx.fragment.app.Fragment
+import com.sunasterisk.youtubebackground.R
+
+class HomeFragment : Fragment() {
+
+    override fun onCreateView(
+        inflater: LayoutInflater, container: ViewGroup?,
+        savedInstanceState: Bundle?
+    ): View? {
+        return inflater.inflate(R.layout.fragment_home, container, false)
+    }
+
+    companion object {
+        fun newInstance(): HomeFragment {
+            return HomeFragment()
+        }
```

Comment:
```suggestion
        fun newInstance()= HomeFragment ()
```
https://kotlinlang.org/docs/reference/coding-conventions.html#function-formatting